package.json

```diff
@@ -29,17 +29,24 @@
   "homepage": "https://github.com/iterative/dvc.org#readme",
   "dependencies": {
     "@octokit/graphql": "^4.3.1",
```

Comment:
do we need to specify engines like we had in blog ? cc @fabiosantoscode

Reply:
cc @iAdramelk

Reply:
Specified it.